src/quo2/foundations/spacing.cljs

```diff
@@ -0,0 +1,16 @@
+(ns quo2.foundations.spacing)
+
+(def get-spacing
```

Comment:
In Clojure, it's more idiomatic to name maps as nouns, not as verbs (even though maps can be used as functions).

Since the namespace is `spacing`, using `spacing/get-spacing` is redundant.

I also don't see the value of using a keyword if it's basically used as an index. I would just use a number. I think that's what @ibrkhalil meant in another comment.

I suggest we change `get-spacing` to just `get` (and remember to use `:refer-clojure` and `:exclude` in case we need the `cljs.core/get` in the same namespace).

```clojure
;; Instead of
(:space-4 spacing/get-spacing)

;; suggestion
(spacing/get 4)
```

--------

But to be honest I don't yet understand this spacing foundation in Figma. It says there the value is a multiplier (of 4 in Figma), but in this PR it's being used as a fixed value. I think it's a good idea to confirm with designers what do they mean by this spacing foundation and **how they use it** in practice because up until this point we haven't used in quo2. @J-Son89, @rahulpsingh

Reply:
Yep I agree @rahulpsingh can you check with design if/how they are using this 👍

Reply:
Yes, the values here should be multiplier(of 4). I will implement it in the suggested way.

Thank you!! 👍

Reply:
This needs some decisions as well and can be done in separate ticket. I just created a ticket for this: https://github.com/status-im/status-mobile/issues/16508

Reply:
Thanks @rahulpsingh !